cooking-user/sj2324-5ehif-cooking-user.Webapi/Startup.cs

```diff
@@ -63,8 +64,8 @@ public async Task Configure(WebApplication app)
         using (var scope = app.Services.CreateScope())
         {
             var services = scope.ServiceProvider;
-            await services.GetRequiredService<UserContext>().Database.EnsureDeletedAsync();
-            await services.GetRequiredService<UserContext>().Database.EnsureCreatedAsync();
+            services.GetRequiredService<UserContext>().Database.EnsureDeleted();
+            services.GetRequiredService<UserContext>().Database.EnsureCreated();
```

Comment:
You can comment these out. I just left these in there for developing purposes when i worked with the dbcontext. Or maybe add a check if environment is development. Because these lines would delete all the stored entries each time which is not desired in production.